osi_object.proto

```diff
@@ -748,132 +761,125 @@ message MovingObject
             //
             // Definition: Hatchback car with maximum length 4 m.
             //
-            // This is one of the OSI types which map from "car" in both OpenScenario 1.x and 2.x.
+            // This is one of the OSI types which map from "car" in both
```

Comment:
How about setting those descriptions into a `\note` statement. This could help to show that OSI does not only "supports" openScenario?

Reply:
Suggestion from the harmonisation group was to remove completely so I've done that.